firebase-functions/functions/src/choiceOperations.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { Player } from './models/player';
```

Comment:
I love the logical structure, easy to read and understand <img src="https://media0.giphy.com/media/26BRv0ThflsHCqDrG/giphy.gif"/>

Reply:
That makes me happy :)